sdk/ml/azure-ai-ml/azure/ai/ml/automl/_automl_image.py

```diff
@@ -41,7 +41,7 @@ def _create_image_job(
     :param target_column_name: The target column name
     :type target_column_name: str
     :param primary_metric: The primary metric
-    :type primary_metric: Optional[Union[str, ClassificationPrimaryMetrics]]
+    :type primary_metric: Optional[Union[str, azure.ai.ml.automl.ClassificationPrimaryMetrics]]
```

Comment:
preface these paths with a "~". It makes sure they get picked up by the Sphinx links.

Reply:
Thanks, Fixed.